### ISSUE TITLE
feature map has black bar?

### ISSUE THREAD

**Issue by mrgreen3325** [shoutOutYangJie/FlowNet1.0-using-Keras#1]
大大，我在轉pytorch weights 去tensorflow 後，發現feature map 的頂端和左辺有黑辺，但pytorch原來的是沒有的，是不是pytorch 和tensorflow padding 方法不同所致？試了很多方法但黑辺还有，你有沒有遇到過？

[photo](https://i.stack.imgur.com/OXkm6.png)

The pytorch code of conv2d is

`nn.Conv2d(1, 56, kernel_size=5, padding=5//2) `
The tensorflow build is

`tensorflow.keras.layers.Conv2D(56, 5, padding="same")(inp) `
I even add the zeropadding layer to the tensorflow model

```
x = tensorflow.keras.layers.ZeroPadding2D(padding=((2,2)))(inp)
x2 = tensorflow.keras.layers.Conv2D(56, 5, padding="valid")(x)
```
However, the black bar is still there. I have no clue for this problem > <. For the converting process for pytorch weights to tensorflow weights is as follow:

```
onnx_1_w_num = onnx_l.weight.data.permute(2,3,1,0).numpy()
onnx_1_b_num = onnx_l.bias.data.numpy()
tf_l.set_weights([onnx_1_w_num,onnx_1_b_num])
```
